spanner/tests/system/test_system.py

```diff
@@ -728,6 +728,40 @@ def _unit_of_work(transaction, test):
 
         return session, committed
 
+    def test_read_with_multiple_keys_index(self):
+        ROW_COUNT = 10
+        EXTRA_DDL = ['CREATE INDEX name ON contacts(first_name, last_name)']
+        COLUMNS = self.COLUMNS[1], self.COLUMNS[2]
+        pool = BurstyPool()
+
+        temp_db = Config.INSTANCE.database(
+            'test_read_multi_keys_index',
+            ddl_statements=DDL_STATEMENTS + EXTRA_DDL,
+            pool=pool
+        )
+        if not temp_db.exists(): # prevent Conflicts 
+            operation = temp_db.create()
+            self.to_delete.append(_DatabaseDropper(temp_db))
+            operation.result(90)
```

Comment:
Because quotas prevent us from using per-CI-run instances, this test needs to mangle the database ID with unique_resource_id.

Another, maybe better strategy might be to just add the CRETATE INDEX DDL to the "stock" set in tests/_fixtures.py.